src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      action="/action_page.php"
+      method="post"
+      class="form"
+    >
+      <div class="form__big-icon icon-form">
+        <img
+          class="form__big-img"
+          src="../src/images/Search.svg"
+          alt="icon search"
+        >
+      </div>
+      <label for="bigInput"></label>
+      <input
+        id="bigInput"
```

Comment:
do not use camelCase for attributes
```suggestion
        id="big-input"
```

Reply:
ви жартуєте мені ментор так сказав написати